tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -321,6 +321,8 @@ kbs_k8s_svc_host() {
 			local host
 			host=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="ExternalIP")].address}' -n "$KBS_NS")
 			[ -z "$host"] && host=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="InternalIP")].address}' -n "$KBS_NS")
+			hostname_i=$(hostname -i)
+			[ "$host" = "$hostname_i" ] && host="localhost"
```

Comment:
`[[ $(hostname -I) =~ "$host" ]]` would be IMO safer

Reply:
Will be fixed in the next iteration.